### DISCUSSION TITLE
Collapsible blocks with custom admonitions

### DISCUSSION THREAD

**Discussion by jromal** [squidfunk/mkdocs-material#3853] [Q&A]
Hi all.

I have a topic that may be an issue.

Before opening an issue discussion I wanted to check if it is a known and discussed topic. But I could not find any hint to hit.

- I have created a couple of custom admonitions and work perfectly (color, icon, ...).
- When I use them as collapsible block, the colors work perfectly, but the icon is the standard pencil icon (like when a non existing admonition name is used).

Is this already known? Maybe dependent on admonition custom icons for insiders (that I am not)? Or is this a matter of an issue item?

Many thanks.

Javier.

**Comment by squidfunk** [comment]
It's not a bug, so there's no need to open an issue. In general, if you use the code that is mentioned in [the documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions), custom admonitions should work in all flavors. You might share your customizations, so somebody can look into it.
